internal/host/hostcommands/install_cmd.go

```diff
@@ -305,15 +308,22 @@ func constructHostInstallerArgs(cluster *common.Cluster, host *models.Host, inve
 		}
 	}
 
+	var installationDisk *models.Disk
 	for _, disk := range inventory.Disks {
 		if disk.ID == host.InstallationDiskID {
-			if disk.DriveType == models.DriveTypeMultipath {
-				installerArgs = append(installerArgs, "--append-karg", "root=/dev/disk/by-label/dm-mpath-root", "--append-karg", "rw", "--append-karg", "rd.multipath=default")
-			} else if disk.DriveType == models.DriveTypeISCSI {
-				// Currently only allowed on the OCI platform
-				installerArgs = append(installerArgs, "--append-karg", "rd.iscsi.firmware=1")
-			}
+			installationDisk = disk
+			break
+		}
+	}
+
+	if installationDisk != nil && installationDisk.DriveType == models.DriveTypeMultipath {
+		installerArgs = append(installerArgs, "--append-karg", "root=/dev/disk/by-label/dm-mpath-root", "--append-karg", "rw", "--append-karg", "rd.multipath=default")
```

Comment:
Can you please explain how it's possible for the installation disk to be multipath? AFAIK, multipath is not eligible as an installation disk (we have blocked it), so how could it be selected as the installation disk?

Reply:
This is the existing code, AFAIK, multipath is blocked for iSCSI, but not fiber channel.

Reply:
Right. Thank you :)